### ISSUE TITLE
U2U1015 is not triggered by this code

### ISSUE THREAD

**Issue by wmjordan** [u2uconsult/codeanalyzers#13]
`__RomanDigitValues[i]` is accessed twice in the loop.

````C#
static readonly int[] __RomanDigitValues = { 1, 4, 5, 9, 10, 40, 50, 90, 100, 400, 500, 900, 1000, 4000, 5000, 9000, 10000, 40000 };
static readonly string[] __RomanDigitChars = { "I", "IV", "V", "IX", "X", "XL", "L", "XC", "C", "CD", "D", "CM", "M", "Mv", "v", "Mx", "x", "xl" };

public static string ToRoman(this int value) {
	if (value > 49999 || value < 1) {
		return string.Empty;
	}
	var result = new StringBuilder(8);
	while (value > 0) {
		for (int i = value < 40 ? 5 : value < 400 ? 9 : __RomanDigitValues.Length - 1; i >= 0; i--) {
			if (value >= __RomanDigitValues[i]) {
				value -= __RomanDigitValues[i];
				result.Append(__RomanDigitChars[i]);
				break;
			}
		}
	}
	return result.ToString();
}
````

**Comment by KrisVandermotten** [comment]
For various reasons, the analyzer currently does not consider fields.

**Comment by KrisVandermotten** [comment]
Version 2.11 has been released a few minutes ago. We now analyze static readonly arrays as well, and the above code raises a warning.

**Comment by wmjordan** [comment]
Just downloaded the new release. It was working as expected.
